CIScripts/BuildStage.ps1

```diff
@@ -0,0 +1,20 @@
+. $PSScriptRoot\Common\Job.ps1
+
+if ($Env:COMPONENTS_TO_BUILD -eq "None") {
+    $Job = [Job]::new("Copying ready artifacts instead of building")
+
+    $ArtifactsPath = "\\10.84.12.40\SharedFiles\WindowsCI-Artifacts"
+    if (Test-Path Env:ARTIFACTS_PATH) {
+        $ArtifactsPath = $Env:READY_ARTIFACTS_PATH
+    }
+
+    $OutputRootDirectory = ".\output"
+    $DiskName = [Guid]::newGuid().Guid
+    New-PSDrive -Name $DiskName -PSProvider "FileSystem" -Root $ArtifactsPath
+    #Copy-Item ("$DiskName" + ":\*") -Destination $OutputRootDirectory -Recurse -Container
+    Copy-Item $ArtifactsPath -Destination $OutputRootDirectory -Recurse -Container
+
+    $Job.Done()
+} else {
+    & $PSScriptRoot\Build.ps1
+}
```

Comment:
Please add `exit 0` to the toplevel scripts (less confusing for durable-task plugin)